xbmc/platform/Platform.h

```diff
@@ -23,17 +23,33 @@ class CPlatform
   static CPlatform *CreateInstance();
 
   /**\brief C'tor */
-  CPlatform();
+  CPlatform() = default;
 
   /**\brief D'tor */
-  virtual ~CPlatform();
+  virtual ~CPlatform() = default;
 
   /**\brief Called at an early stage of application startup
    *
    * This method can be used to do platform specific environment preparation
    * or initialisation (like setting environment variables for example)
    */
-  virtual bool Init();
+  virtual bool Init() { return true; };
```

Comment:
might be worth it to update this to `InitStageOne`?

Reply:
I thought the same thing when I saw this line.  I vote for it.

Reply:
I contemplated it, then I said stuff it. Haha. I'll go back and do it though